src/Kis/Kis.hs

```diff
@@ -22,23 +23,46 @@ import Kis.Model
 import Kis.Time
 
 data KisAction a where
-    CreateBed :: String -> KisAction BedId
-    CreatePatient :: Patient -> KisAction PatientId
+    CreateBed :: String -> KisAction (Maybe BedId)
+    CreatePatient :: Patient -> KisAction (Maybe PatientId)
```

Comment:
Der Grund ist, dass ich in der readOnly Umgebung auch etwas zurückgeben muss. Vielleicht könnte man das über eine Exception machen, falls wir es doch brauchen. Ich implementiere es erstmal ohne readOnly Umgebung.